src/OrchardCore.Modules/OrchardCore.Lists/Services/ContainerService.cs

```diff
@@ -283,6 +290,29 @@ public async Task<IEnumerable<ContentItem>> QueryContainedItemsAsync(string cont
                     return containedItems;
                 }
 
+                if (listPartFilter.DisplayText != null)
+                {
+                    containedItems = containedItems.Where<ContentItem>(i => i.ContentItem.As<TitlePart>().Title.Contains(listPartFilter.DisplayText)).ToList();
```

Comment:
Nem a `TitlePart`-ra kellene szűrni, hanem a `DisplayText`-re, ahogy a `DefaultContentsAdminListFilter` is csinálja.

Reply:
Rendben